src/ui/actions/metadata.js

```diff
@@ -181,3 +184,29 @@ export function updateUser(authUser = {}) {
     dispatch({ type: "register_user", user: updatedUser });
   };
 }
+
+function onTitleUpdate(title) {
+  return async ({ dispatch, getState }) => {
+    if (title === selectors.getTitle(getState())) {
+      return;
+    }
+
+    if (!title) {
+      await new Promise(r => setTimeout(r, 5000));
+      const titleVal = await ThreadFront.evaluate(0, null, "window.location.host");
```

Comment:
We'll be able to replace this in the future when we share data with the recordings app.